### DISCUSSION TITLE
Codespace -"setting up your codespace" is popping

### DISCUSSION THREAD

**Discussion by prajwalgowdasa** [community/community#126939] [New to GitHub]
I am trying open my first codespace in GitHub but it just showing a blank page with "Setting up your codespace "
Written in it

**Comment by YuheshPandian** [comment]
try deleting that codespace that creating another and update the status for further assistance.

**Comment by prajwalgowdasa** [comment]
It worked thanks a lot.

**Reply by YuheshPandian** [reply]
Your Welcome
